### ISSUE TITLE
I can haz velocity?  (km/s)

### ISSUE THREAD

**Issue by davidmikolas** [skyfielders/python-skyfield#136]
I'd like to get state vectors.  Is there a best/correct way to get the velocity in km/s or similar unit without looking up AU in WIkipedia and typing the conversion in somewhere? 

    import numpy as np
    from skyfield.api import load
    data  = load('de421.bsp')
    ts    = load.timescale()
    t     = ts.utc(2017, 5, 12, 0)
    earth = data['earth']
    print earth.at(t).position          # ok
    print earth.at(t).position.km       # ok
    print earth.at(t).velocity          # ok
    print earth.at(t).velocity.km       # fails
    print earth.at(t).velocity.kms      # etc.

![image](https://cloud.githubusercontent.com/assets/17236804/26033051/310255a8-38d5-11e7-8936-5d3e14acef69.png)

http://i.imgur.com/Q9Pxc0x.jpg

**Comment by oefe** [comment]
Try `km_per_s `.

**Comment by davidmikolas** [comment]
I CAN haz velocity!  Thanks!

How would one find that out?

Also, please feel free to close this issue if you like.

    >>> if 1 == 1:
    ...     import numpy as np
    ...     from skyfield.api import load
    ...     data = load('de421.bsp')
    ...     ts   = load.timescale()
    ...     t    = ts.utc(2017, 5, 12, 0)
    ...     earth = data['earth']
    ...     print earth.at(t).position
    ...     print earth.at(t).position.km
    ...     print earth.at(t).velocity
    ...     print earth.at(t).velocity.km_per_s
    ...
    [-0.62957002 -0.71815183 -0.31148123] au
    [ -9.41823344e+07  -1.07433984e+08  -4.65969284e+07]
    [ 0.01312422 -0.00993451 -0.00430658] au/day
    [ 22.72402816 -17.20118237  -7.45665504]

**Comment by oefe** [comment]
> How would one find that out?

By reading the [source (units.py)](https://github.com/skyfielders/python-skyfield/blob/master/skyfield/units.py) ;-)

Or, as the native unit is `au_per_d`, one might guess that km/s is using the same format, hence `km_per_s`.

This could certainly be improved. In order to optimize memory use, Skyfield initially only stores the value in the "native" unit, and calculates other units only when needed, but then caches them to speed up future access. It currently uses a homegrown implementation to implement those "cached properties". Maybe use [`cached-property`](https://pypi.python.org/pypi/cached-property) instead?

Or, document alternate units in the class' docstring.

**Comment by davidmikolas** [comment]
Interesting points.

In my case I knew to type `.km` because I remembered from an earlier version, and I thought I remembered that `.km` applied to a velocity also worked back then. 

When it didn't work, I landed in the documentation here: http://rhodesmill.org/skyfield/api-units.html#skyfield.units.Velocity which just mentions "other units" but not which ones or how, and of course, `dir()` does not show these methods.

If I had gone to the main API page first http://rhodesmill.org/skyfield/api.html I would have seen the `.km_per_s`
